### DISCUSSION TITLE
Bug on GitHub Mobile

### DISCUSSION THREAD

**Discussion by SlaVcE14** [community/community#60938] [Mobile] [answered]
When you refreshing the discussion page on GitHub Mobile the list is glitching

https://github.com/community/community/assets/102253539/5d74a0ae-b410-42d6-8631-b99df0405657

**Comment by esahmed21** [comment, marked as answer]
Hi @SlaVcE14  👋

Thank you for reaching out to us! That definitely should not happen, I'll pass your feedback to the team and make sure to track this bug and fix it.

Let me know if we can help with anything else. 
Thanks!

**Comment by ettaboyle** [comment]
Hi @SlaVcE14 👋🏻  Thank you for marking this post as answered! I'm going to go ahead and "re-open" this discussion so if another user has the same issue or sees this bug they will see the feedback from @esahmed21 and not start a new discussion.

_FYI: When discussions are "closed" they often won't show up in search so re-opening this will hopefully help others._

Thank you!

**Comment by YMANBEKO** [comment]
> When you refreshing the discussion page on GitHub Mobile the list is glitching
> 
> https://github.com/community/community/assets/102253539/5d74a0ae-b410-42d6-8631-b99df0405657
> 
>
